spring-kafka/src/main/java/org/springframework/kafka/annotation/EnableKafka.java

```diff
@@ -249,6 +250,6 @@
 @Target(ElementType.TYPE)
 @Retention(RetentionPolicy.RUNTIME)
 @Documented
-@Import(KafkaBootstrapConfiguration.class)
+@Import({KafkaBootstrapConfiguration.class, KafkaBootstrapConfiguration.KStreamsRegistrar.class})
```

Comment:
Maybe we should have a separate annotation for `@EnableKafkaStreams`? - it will rarely be a case that you'd have both a Kafka Stream and producer/consumers in the same app. Also, for regular Kafka apps I guess this also starts a Stream with an empty topology (probably wouldn't do much but I guess we can avoid it).